ui/src/features/WorkspaceJobs/components/JobDetails/hooks.ts

```
@@ -12,6 +13,14 @@ export default ({ jobId }: { jobId: string }) => {
 
   const { useGetJob, useJobCancel } = useJob();
 
+  const { data: jobStatus, isLoading, error } = useJobStatus(jobId);
+
+  const statusValue = isLoading
+    ? t("Loading...")
+    : error
+      ? t("Error")
+      : jobStatus || job.status;
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)